docs/central-upgrade.rst

```diff
@@ -25,65 +26,351 @@ Upgrade notes
+   
+       #. **Check the output of the previous command to see if there were any errors.** If there were any errors that you can't resolve, `write a support post on the forum <https://forum.getodk.org/c/support/6>`_.
+   
+       #. **Check the upgrade success file has been created.**
+   
+          .. code-block:: console
+   
+             $ ls ./files/postgres14/upgrade/upgrade-successful
+   
+          If you see "No such file or directory," try doing ``docker compose up postgres`` again. If the file has still not been created, `write a support post on the forum <https://forum.getodk.org/c/support/6>`_.
+   
+       #. **Restart the server.**
+   
+          .. code-block:: console
+   
+               $ docker compose up -d
+   
+       #. **Log into the web interface and do some quick spot checks.** For example, verify that submission counts and latest submission dates look right and try a data export.
+   
+       #. **Clean up**
+   
+          #. **Remove unused Docker images**.
+
+             .. code-block:: console
+
+                 $ docker image prune
+
+             You'll be asked to confirm the removal of all dangling images. Agree by typing ``y`` and pressing Enter.
+
+          #. **See how much space the old database takes**. The upgrade process performs a copy and leaves the old database intact.
+
+             .. code-block:: console
+
+                  $ docker compose up postgres
+
+          #. **Delete the old data**. Make sure you have verified that the server works as expected first.
+
+             .. code-block:: console
+   
+               $ touch ./files/postgres14/upgrade/delete-old-data \
+                  && docker compose up --abort-on-container-exit postgres
+   
+     .. tab:: Custom database
+   
+       #. **Find instructions for upgrading your database server to PostgreSQL 14**. Here are instructions for some popular fully-managed options:
+   
+          * `DigitalOcean <https://docs.digitalocean.com/products/databases/postgresql/how-to/upgrade-version/>`_
+          * `Amazon <https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_UpgradeDBInstance.PostgreSQL.html#USER_UpgradeDBInstance.PostgreSQL.MajorVersion.Process>`_
+          * `Azure <https://learn.microsoft.com/en-us/azure/postgresql/single-server/how-to-upgrade-using-dump-and-restore>`_
+   
+       #. **Determine whether upgrading your database requires downtime**. If it does, stop Central before continuing:
+   
+          .. code-block:: console
+   
+            $ docker compose stop
+   
+       #. **Upgrade your database server**. We recommend using the latest point release of PostgreSQL 14 that is available.
+
+       #. **Create a file to prove that you're carefully reading these instructions.** This is required to continue.
+   
+          .. code-block:: console
+   
+            $ touch ./files/allow-postgres14-upgrade
+   
+       #. Follow the :ref:`standard upgrade instructions <central-upgrade-steps>`.
```

Comment:
This could totally stay as a link to the one place the instructions are written, but I was _thinking_ what if there was a little summary of the instructions below:

```
cd central (they've probably already done this)
git pull
git submodule update -i
docker compose build
docker image prune
docker compose up -d
```

Reply:
The standard instructions are concise and copy-able enough now that I'd rather send them there so they can see any unexpected output step by step.